aws/templates/id/id_ecs.ftl

```diff
@@ -27,6 +26,12 @@
                 component)]
 [/#function]
 
+
+[#-- Components --]
+[#assign ECS_COMPONENT_TYPE = "ecs" ]
+[#assign SERVICE_COMPONENT_TYPE = "service" ]
```

Comment:
I think maybe subcomponent types should include the parent e.g. ECS_SERVICE_COMPONENT_TYPE?